build_compilers/lib/src/workers.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:bazel_worker/driver.dart';
+import 'package:build/build.dart';
+import 'package:cli_util/cli_util.dart' as cli_util;
+import 'package:path/path.dart' as p;
+
+String get _scriptExtension => Platform.isWindows ? '.bat' : '';
+
+const _defaultMaxWorkers = 4;
```

Comment:
Maybe base this on a factor of the number of cores, which is readily available through the Platform API.

Reply:
done